### PR TITLE
Update try-runtime.sh

### PR DESCRIPTION
**Pull request by mordamax**
Fix: https://github.com/paritytech/command-bot/issues/145

Tested here: https://github.com/paritytech-stg/polkadot/pull/2#issuecomment-1364658239

### CODE REVIEW

File: commands/try-runtime/try-runtime.sh [hunk 1]
```diff
@@ -20,27 +26,25 @@ main() {
...
-    --uri wss://${network}-try-runtime-node.parity-chains.parity.io:443
-  )
-
   set -x
   export RUST_LOG="${RUST_LOG:-remote-ext=debug,runtime=trace}"
-  cargo "${preset_args[@]}" "$@"
+
+  # following docs https://paritytech.github.io/substrate/master/try_runtime_cli/index.html
+  cargo build --release
+  cp "./target/release/${repository}" .
```
Comment (kianenigma) [inline commands/try-runtime/try-runtime.sh:34]:
```suggestion
```

Comment (kianenigma) [reply commands/try-runtime/try-runtime.sh:34]: is being done in line 38 as well?

File: commands/try-runtime/try-runtime.sh [hunk 2]
```diff
@@ -20,27 +26,25 @@ main() {
...
+  cp "./target/release/${repository}" node-try-runtime
+  cp "./target/release/wbuild/${network}-runtime/${network}_runtime.wasm" runtime-try-runtime.wasm
+
+  ./node-try-runtime \
+    try-runtime \
+    --runtime runtime-try-runtime.wasm \
+    -lruntime=debug \
+    on-runtime-upgrade \
+    live --uri "wss://${network}-try-runtime-node.parity-chains.parity.io:443" \
+    "$@"
```
Comment (kianenigma) [inline commands/try-runtime/try-runtime.sh:47]: might be neat to remove all artifacts now? not sure if it is needed. is the build env persisted in any way?

**Review by kianenigma — APPROVED** [review]
hallelujah!

File: commands/try-runtime/try-runtime.sh (hunk 1 above)
Comment (mordamax) [reply commands/try-runtime/try-runtime.sh:34]: ah, this was from docs, but indeed, not sure why this files was copied 
image

**Pull request merged**